lua/tailwindcss-colorizer-cmp/init.lua

```diff
@@ -84,12 +84,28 @@ M.formatter = function(entry, vim_item)
 			table.insert(words, word)
 		end
 
+		local color_name, color_number
+		if
+			words[2] == "x"
+			or words[2] == "y"
+			or words[2] == "t"
+			or words[2] == "b"
+			or words[2] == "l"
+			or words[2] == "r"
+		then
+			color_name = words[3]
+			color_number = words[4]
+		else
+			color_name = words[2]
+			color_number = words[3]
+		end
+
 		-- FIXME:
 		-- handle special case of stuff like whatever-black/white
 		-- are there other cases like foo-bar-white
```

Comment:
Pretty sure this comment isn't relevant any more, remove?
```suggestion
```

Reply:
Yep, please remove!